.github/test-spec.yml

```diff
@@ -188,6 +188,7 @@
   - "lib/date_time/**/*"
   - "drivers/sensor/sensor_sim/**/*"
   - "applications/asset_tracker_v2/**/*"
+  - "applications/asset_tracker_v2/tests/**/*"
```

Comment:
The line above should cover the tests directory too right?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/8100#discussion_r991046410